neuralmonkey/config/parsing.py

```diff
@@ -143,84 +142,49 @@ def parse_value(string):
 
     return string
 
+def parse_config(config_file, filename=""):
```

Comment:
Před mergem bych potřeboval vědět, jestli to je nebo neni zpětně kompatibilní. Moc nechápu proč by nebylo a o to víc mě to "zatímco tamto by mělo bejt zpětně kompatibilní" straší.